03week/ticTacToe.js

```diff
@@ -23,25 +32,109 @@ function printBoard() {
   console.log('2 ' + board[2].join(' | '));
 }
 
-function horizontalWin() {
-  // Your code here
+//rules for every horizontal win
+const horizontalWin=()=> {
+  console.log('playerTurn=',playerTurn)
+  if(board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn) {
```

Comment:
you're repeating yourself a lot here, just return the evaluation. How else could you simplify?